src/test/resources/application-test.yml

```diff
@@ -16,6 +16,8 @@ spring:
     url: jdbc:postgresql://localhost:5432/restricted-patients
     username: restricted-patients
     password: test
+  flyway:
+    clean-disabled: false
```

Comment:
needed as default now not to allow a clean to prevent people destroying production databases

Reply:
:-)